frontend/src/components/form/CopyToClipboard.vue

```diff
@@ -21,11 +21,7 @@ const toast = useToast();
 const copyToClipboard = () => {
   if( props.toCopy ) {
     navigator.clipboard.writeText(props.toCopy);
-    toast.add({
-      severity: 'info',
-      summary: 'Copied to clipboard',
-      life: 3000,
-    });
+    toast.info('Copying clipboard', 'Copied to clipboard');
```

Comment:
Style: This seems a bit strange the way its represented. It should be sufficient to just say `Copied to clipboard` as the header instead of having a present-progressive and a past participle statement in one toast. Would recommend consulting with Tyler on this if this hasn't been raised.